ls1intum/Artemis
# `Lectures`: Fix inconsistent delete dialog for online unit

### Checklist
#### General


#### Client


### Motivation and Context


Closes #6014 

### Description

Previously, the dialog to delete an online unit was different from all the other ones and didn't require the user to enter the name of the unit again before being able to delete it. This PR fixes this.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Course

1. Log in to Artemis
2. Navigate to Course Administration
3. Navigate to Lectures
4. Create all kinds of lecture units and delete them again. The dialogs should be consistent now.

### Review Progress






#### Code Review
#### Manual Tests

### Test Coverage





### Screenshots


Before:
![image](https://user-images.githubusercontent.com/84102468/210348204-4e2ab8de-e136-42b5-9afe-704475a21d82.png)

After:
![image](https://user-images.githubusercontent.com/84102468/210347935-1c93d517-562d-4015-b30e-e6a7ac5038cf.png)
